x/posts/client/rest/rest.go

```diff
@@ -18,19 +18,20 @@ import (
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, storeName string) {
 	r.HandleFunc("/posts", createPostHandler(cliCtx)).Methods("POST")
 	r.HandleFunc("/posts/{postID}", getPostHandler(cliCtx, storeName)).Methods("GET")
-	r.HandleFunc("/posts/likes", likePostHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/posts/reactions", addReactionToPostHandler(cliCtx)).Methods("POST")
 	r.HandleFunc("/posts/likes", unlikePostHandler(cliCtx)).Methods("DELETE")
```

Comment:
Should we change this endpoint to `/reactions` as well?

Reply:
Thanks, also renamed the handler